site/cgi-bin/xlsx_to_csv.cgi

```diff
@@ -25,10 +25,11 @@ def main():
     print("Content-type: text/html")
     print()
 	# Do a localhost check
-	ip_addr = cgi.escape(os.environ["REMOTE_ADDR"])
-	if ip_addr != "127.0.0.1" and ip_addr != "localhost":
-		print_error("Cannot be called from external host")
-		return
+	# TODO : FIX THIS, IT IS CRASHING THE SCRIPT
+	#ip_addr = cgi.escape(os.environ["REMOTE_ADDR"])
+	#if ip_addr != "127.0.0.1" and ip_addr != "localhost":
+	#	print_error("Cannot be called from external host")
+	#	return
```

Comment:
This whole block can be removed and there's no point trying to "fix it". We've already secured this script in a better, more robust fashion to do this through apache.

Reply:
Done